buildlib/azure-pipelines-perf.yml

```diff
@@ -10,6 +10,8 @@ pr:
 variables:
 - name: WorkDir
   value: /hpc/scrap/azure/$(Build.DefinitionName)/$(Build.BuildId)-$(Build.BuildNumber)
+- name: warning
+  value: Attention! CI testing is in progress. Please refrain from any manual action on this machine.
```

Comment:
Maybe instead of warning configure the setup to prohibit unprivileged user access and change default root creds?

Reply:
We do want users to connect and use the environment for manual repro attempts etc.